tests/integration/targets/vmware_ops_vcenter_host_connection_test/tasks/main.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Test
+  block:
+    - name: Add Resource Pool
+      ansible.builtin.include_tasks: call_resource_pool_role.yml
+      vars:
+        required_state: present
+      tags: eco-vcenter-ci
+
+    - name: Provision Virtual ESXI and set ESXI hostname
+      ansible.builtin.include_tasks: provision_esxi.yml
+      tags: eco-vcenter-ci
+
+    - name: Add ESXI Host
+      ansible.builtin.import_role:
+        name: cloud.vmware_ops.vcenter_host_connection
+      vars:
+        vcenter_host_connection_state: present
+
+    - name: Remove ESXI Host
+      ansible.builtin.import_role:
+        name: cloud.vmware_ops.vcenter_host_connection
+      vars:
+        vcenter_host_connection_state: absent
+
+  always:
+    - name: Remove ESXI Host to cleanup env
+      community.vmware.vmware_host:
+        hostname: "{{ vcenter_hostname }}"
+        username: "{{ vcenter_username }}"
+        password: "{{ vcenter_password }}"
+        datacenter: "{{ vcenter_datacenter }}"
+        cluster: "{{ cluster_name }}"
+        validate_certs: false
+        esxi_hostname: '{{ vcenter_host_connection_esxi_host | default("") }}'
+        esxi_username: '{{ esxi_username }}'
+        esxi_password: '{{ esxi_password }}'
+        state: absent
+      tags: eco-vcenter-ci
```

Comment:
maybe we can use the role itself here? I mean - `cloud.vmware_ops.vcenter_host_connection`

Reply:
I thought it better to use community role, because role itself under the test and one of the reasons why this task should be used is that 'Remove ESXI Host' task from 'vcenter_host_connection' role failed.

Reply:
ahh I see what you mean now, ok, that sounds smarter :)